src/services/exposed/error-handler.js

```diff
@@ -14,12 +14,16 @@ function errorHandler({ logger } = {}) {
   return function handleError(error, request, response, next) {
     if (error) {
       // NOTICE: Send the first error if any.
-      if (error.errors && error.errors[0] && error.errors[0].message) {
-        error.message = error.errors[0].message;
-      }
-
-      if (error.errors && error.errors[0] && error.errors[0].name) {
-        error.name = error.errors[0].name;
+      if (error.errors && error.errors[0]) {
```

Comment:
I'm not sure to understand this use case on the agent side. Is it required due to `VError` usages?